Assets/MixedRealityToolkit.Services/InputSimulation/MixedRealityInputSimulationProfile.cs

```diff
@@ -87,30 +112,34 @@ public class MixedRealityInputSimulationProfile : BaseMixedRealityProfile
         [Header("Hand Simulation")]
         [SerializeField]
         [Tooltip("Enable hand simulation")]
-        private HandSimulationMode handSimulationMode = HandSimulationMode.Articulated;
-        public HandSimulationMode HandSimulationMode { get => handSimulationMode; set => handSimulationMode = value; }
+        private HandSimulationMode defaultHandSimulationMode = HandSimulationMode.Articulated;
+        public HandSimulationMode DefaultHandSimulationMode => defaultHandSimulationMode;
 
         [Header("Hand Control Settings")]
         [SerializeField]
         [Tooltip("Key to toggle persistent mode for the left hand")]
-        private KeyCode toggleLeftHandKey = KeyCode.T;
-        public KeyCode ToggleLeftHandKey => toggleLeftHandKey;
+        private KeyBinding toggleLeftHandKey = KeyBinding.FromKey(KeyCode.T);
+        public KeyBinding ToggleLeftHandKey => toggleLeftHandKey;
         [SerializeField]
         [Tooltip("Key to toggle persistent mode for the right hand")]
-        private KeyCode toggleRightHandKey = KeyCode.Y;
-        public KeyCode ToggleRightHandKey => toggleRightHandKey;
+        private KeyBinding toggleRightHandKey = KeyBinding.FromKey(KeyCode.Y);
+        public KeyBinding ToggleRightHandKey => toggleRightHandKey;
         [SerializeField]
         [Tooltip("Time after which uncontrolled hands are hidden")]
         private float handHideTimeout = 0.2f;
         public float HandHideTimeout => handHideTimeout;
         [SerializeField]
         [Tooltip("Key to manipulate the left hand")]
-        private KeyCode leftHandManipulationKey = KeyCode.LeftShift;
-        public KeyCode LeftHandManipulationKey => leftHandManipulationKey;
+        private KeyBinding leftHandManipulationKey = KeyBinding.FromKey(KeyCode.LeftShift);
+        public KeyBinding LeftHandManipulationKey => leftHandManipulationKey;
         [SerializeField]
         [Tooltip("Key to manipulate the right hand")]
-        private KeyCode rightHandManipulationKey = KeyCode.Space;
-        public KeyCode RightHandManipulationKey => rightHandManipulationKey;
+        private KeyBinding rightHandManipulationKey = KeyBinding.FromKey(KeyCode.Space);
+        public KeyBinding RightHandManipulationKey => rightHandManipulationKey;
+        [SerializeField]
+        [Tooltip("Controls how mouse look control is activated")]
+        private KeyBinding handRotateButton = KeyBinding.FromMouseButton(KeyBinding.MouseButton.Right);
```

Comment:
>private KeyBinding handRotateButton = KeyBinding.FromMouseButton(KeyBinding.MouseButton.Right); [](start = 8, length = 95)

What are your thoughts on adding a comment regarding the priority order of hand rotate vs camera rotate? Otherwise the question here is around "hey, both the hand and camera have the same mouse binding" - AFAICT the camera wins out if both try to be active.

Reply:
Ah wait, it's the other way around.

---
In reply to: [320511622](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5853#discussion_r320511622) [](ancestors = 320511622)

Reply:
I will change the default key binding to Ctrl for the hand rotation, so (by default) there won't be a conflict. And then if you set them to the same binding again it will just do both camera and hand rotation at the same time.

My thinking originally was that it would be better to just have one key for enabling rotation for anything, but it seems that the combinations are a bit fiddly, and by having dedicated keys for both rotations it just extends the current system and doesn't mess with muscle memory so much. There's just no ideal solution here, just more or less annoying tradeoffs ...